### ISSUE TITLE
Add formatter for html

### ISSUE THREAD

**Issue by lookupdaily** [dxw/playbook#1192]
Currently we have no formatter in place for html files. We could add Prettier for this, but would need to make sure it does not conflict with JavaScript Standard

This would help us to keep our code consistent when editing html files, and reduce formatting changes on changes to existing files (if a developer uses a formatter locally).

**Comment by yndajas** [comment]
Could we just tell it to ignore `.js` files? Or alternatively switch to Prettier wholesale and drop Standard?

**Comment by lookupdaily** [comment]
Yep, that's how we've done it on our current project (ignoring js files). If we binned standard, we'd need to use eslint anyway, as they have [separate concerns](https://prettier.io/docs/en/comparison), however standard covers formatting as well as rule checking i believe?  

Prettier has [guidance on integrating with eslint](https://prettier.io/docs/en/integrating-with-linters), but not standard.
